src/frontend/test_runner/tests/testdata/tpch.yaml

```diff
@@ -1218,44 +1218,34 @@
   before:
     - create_tables
   sql: |
-    select
-      s_suppkey,
-      s_name,
-      s_address,
-      s_phone,
-      total_revenue
-    from
-      supplier,
-      (
-        select
+    with revenue0 (supplier_no, total_revenue) as (
```

Comment:
Just simplifying the test case with newly-supported cte. Not related to the fix.